blog/_posts/2021-05-04-effective-programming-scala-course.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy
+title: "New Online Course: Effective Programming in Scala"
+---
+
+We are excited to announce that our new MOOC “Effective Programming in Scala” is now
+live on Coursera! We are also happy to share that most of our existing courses have already
+been updated to Scala 3.
+
+## Effective Programming in Scala
+
+The [Effective Programming in Scala] course
+is 6 weeks long, and aims to teach non-Scala programmers everything they need to become
```

Comment:
Phrasing it like this makes it feel to me that the '6 weeks long' part is the most important. I think that should be mentioned later.

Reply:
I agree on this feeling.
You could lighten with 
```suggestion
aims to teach non-Scala programmers everything they need to become professional Scala 3 programmers in six weeks.
```

Reply:
I agree. I have included Vince’s suggestion.